### DISCUSSION TITLE
favourite/like button?

### DISCUSSION THREAD

**Discussion by Retrofreak** [rern/rAudio#821] [Feature Requests]
Sometimes there's suddenly a song, which I forgot and I don't want to forget it again ;-)  A little heart, star, or something similar would be great. After clicking it, the current playing tune is added to a favourites-playlist. Possible?

**Comment by rern** [comment]
- Playlist > Context menu > Add to saved playlist
- Select target > Select position to insert - first, select or last.

There must be at least 1 already saved otherwise save the current one for a start.

**Reply by Retrofreak** [reply]
Okay, thanks - works. Although I was looking for a more simple way: I just want to touch a little icon on the playback-screen and the currently playing song is added to a predefined playlist?
